packages/serve/src/index.ts

```diff
@@ -132,6 +132,40 @@ class ServeCommand {
 
         const servers = [];
 
+        const stopAllServers = () => {
+          Promise.all(
```

Comment:
I'd love to make this a variable and passing the variable to the promise expression